torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -632,15 +634,15 @@ at::Tensor interpolate(
     return at::upsample_nearest2d(
         input,
         _output_size(input, 2, size, scale_factors),
-        scale_factors_1,
-        scale_factors_2);
+        c10::make_optional(scale_factors_1),
```

Comment:
I need to explicitly put scale factor as c10::optional otherwise composite [`at::upsample_nearest2d`](https://github.com/pytorch/pytorch/pull/97868/files#diff-5235eb89d3bbcfdbf84c8ad0f96b2c309f3ccb542501b26946a0a47becb7d3eaR150) is called with double scale factor interpreted as bool for `round_with_scale_factor`.